### DISCUSSION TITLE
IOS dedicated app ?

### DISCUSSION THREAD

**Discussion by Mrdindon** [arendst/Tasmota#11673] [Ideas]
I'm surprised there is still no one who developed an IOS app for Tasmota... I agree Homebridge exist but that's just an additional layer between the devices and the phone so I was wondering if someone here already started working on a UI for IOS ?

I saw the Yeti app a while ago but I can't install it from the US/Canadian/French appstore market.

Cheers !

**Comment by NsinghP** [comment]
I have been using the home remote app (Free) on my iphone to locally control fans and light in my home ( https://thehomeremote.com/ ). I am using tasmota on ESP8266-12F based device that has hue emulation on by default.

  PS: I can also control lights and fans remotely using Alexa/echo dot from anywhere in the world with internet access.

**Comment by barbudor** [comment]
Any App that supports MQTT can be integrated with a home system based on MQTT and sends command to Tasmota.
But access to an in-house infrastructure from outside of your home network may be a security challenge.

**Comment by sfromis** [comment]
The general expectation is that you do not use an app to directly handle Tasmota devices, but have a backend dashboard doing that. This may then have apps for IOS and so on, like when it appears (no personal experience) that Home Assistant has an IOS app.
